blogposts/2021/release-3.31.md

```diff
@@ -0,0 +1,77 @@
+---
+title: 'Sourcegraph 3.31 release'
+publishDate: 2021-09-01T10:00-07:00
+description: 'Sourcegraph 3.31 introduces the Private Beta release of Code Insights, complete with no repository limits plus repository-based filters. It also introduces automated migration of saved search notifications to code monitoring.'
+tags: [blog, release]
+slug: 'release/3.31'
+published: true
+heroImage: https://sourcegraphstatic.com/blog/3.31/sourcegraph-3-31-release.png
+socialImage: https://sourcegraphstatic.com/blog/3.31/sourcegraph-3-31-release.png
+changelogItems:
+  - description: 'Backend Code Insights GraphQL queries now support arguments `includeRepoRegex` and `excludeRepoRegex` to filter on repository names.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/23256
+    category: Code Insights
```

Comment:
Users won't be using the backend GraphQL queries, so this doesn't feel very relevant to the reader.

```suggestion
```